### DISCUSSION TITLE
Improved loadingOverlay support  as a wrapper

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4428] [Feature requests]
https://ant.design/components/spin#components-spin-demo-nested

```jsx
<LoadingOverlay loading={true}>
...
</LoadingOverlay>
```

**Comment by bogordesaincom** [comment]
Hey, you can change 

```
<LoadingOverlay visible={true}>
...
</LoadingOverlay>
```
Please see props 

https://mantine.dev/core/loading-overlay/?t=props

Or User Loader for wrapper

```
<Loader />
```
https://mantine.dev/core/loader/

**Reply by vaynevayne** [reply]
for docs, where   Mention  LoadingOverlay support children?
